x-pack/plugins/ml/public/data_frame/components/job_create/job_create_form.tsx

```diff
@@ -230,17 +237,17 @@ export const JobCreateForm: SFC<Props> = React.memo(
                   'xpack.ml.dataframe.jobCreateForm.createAndStartDataFrameDescription',
                   {
                     defaultMessage:
-                      'Create and start the data frame job. After the job is started, you will be offered options to continue exploring the data frame job.',
+                      'Use with caution, creating a data frame job may cause a significant load on your cluster. After the job is started, you will be offered options to continue exploring the data frame job.',
                   }
                 )}
               </EuiText>
             </EuiFlexItem>
           </EuiFlexGroup>
         )}
         {created && (
-          <EuiFlexGroup alignItems="center" style={{ maxWidth: '800px' }}>
-            <EuiFlexItem grow={false} style={{ width: '200px' }}>
-              <EuiButton isDisabled={created && started} onClick={startDataFrame}>
+          <EuiFlexGroup alignItems="center" style={FLEX_GROUP_STYLE}>
+            <EuiFlexItem grow={false} style={FLEX_ITEM_STYLE}>
+              <EuiButton color="warning" isDisabled={created && started} onClick={startDataFrame}>
```

Comment:
As discussed, reverted the button back to filled blue but decided to not show a modal because we have the additional text here in constrast to the jobs list.